cmd/protoc-gen-crd/openapiGenerator.go

```diff
@@ -299,7 +306,11 @@ func (g *openapiGenerator) generateFile(
 	// Name -> CRD
 	crds := map[string]*apiext.CustomResourceDefinition{}
 
-	for name, cfg := range genTags {
+	for name, cfg := range messageGenTags {
+		if cfg["releaseChannel"] == "standard" && !includeExperimental {
```

Comment:
Should this be checking if the "releaseChannel" is `stable`?

Reply:
Only `standard` resources and fields will have a `releaseChannel` tag. So this is checking to see if the resource has a tag `releaseChannel:standard`, then if `includeExperimental` is `false` skip the generation of the standard resource in the stable channel.